test/protocols/autoTestGAPResults.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+
+#MOLGENIS walltime=05:59:00 mem=8gb ppn=1
+
+#string tmpName
+#string	Project
+#string logsDir
+#string intermediateDir
+
+set -e
+set -u
+
+module load HTSlib
+module load BEDTools
+module load ngs-utils
+module list
+
+## Filtering of the VCF files from the pipeline
+
+#1 Compress the VCF file
+
+
+for i in  $(ls /groups/umcg-gsad/tmp03/projects/NIST_TRIO/run01/results/vcf/*.vcf)
+do
+	file=$(basename ${i})
+	mkdir -p /groups/umcg-gsad/tmp03/projects/NIST_TRIO/temp/
+	echo  "zipping vcf file : ${file} ..."
+	bgzip -c ${i} > /groups/umcg-gsad/tmp03/projects/NIST_TRIO/temp/${file}.gz
+
+done
+
+#2 making the index of the VCF file
+
+for i in $(ls /groups/umcg-gsad/tmp03/projects/NIST_TRIO/temp/*.gz)
+do
+	echo "indexing the vcf file of ${i} ..."
+	tabix -p vcf ${i}
+done
+
+#3 Filtering the VCF using the bedfile
+
+for i in $(ls /groups/umcg-gsad/tmp03/projects/NIST_TRIO/temp/*.gz)
+do
+	echo "filtering the vcf file: ${i} ..."
+	file=$(basename ${i})
+	sample=$(basename ${file} .FINAL.vcf.gz)
+	bedtools intersect -a ${i} -b /home/umcg-molgenis/GAP/autoTestArray.bed > /groups/umcg-gsad/tmp03/projects/NIST_TRIO/temp/${sample}.FINAL_FILTERED.vcf
+done
+
+## Comparing the VCF files
+
+for i in $(ls /groups/umcg-gsad/tmp03/projects/NIST_TRIO/temp/*.FINAL_FILTERED.vcf)
+do
+	file=$(basename ${i})
+	echo ${file}
+	sample=$(basename ${file} .FINAL_FILTERED.vcf)
+	echo "Comparing the VCF file with the TRUE VCF for the sample sample : ${sample} ..."
+	${EBROOTNGSMINUTILS}/vcf-compare_2.0.sh -1 "/home/umcg-molgenis/GAP/vcf/${sample}.FINAL_TRUE_FILTERED.vcf" -2 "${i}" -o "/groups/umcg-gsad/tmp03/projects/NIST_TRIO/temp/VCF_Compare/${sample}/"
+done
+
+## Checking if the output is correct
+
+for i in $(ls /groups/umcg-gsad/tmp03/projects/NIST_TRIO/temp/*.FINAL_FILTERED.vcf)
+do
+	file=$(basename ${i})
+	echo ${file}
+	sample=$(basename ${file} .FINAL_FILTERED.vcf)
+	vcfCheckvalue="$(awk 'NR == 2 {print $4}' /groups/umcg-gsad/tmp03/projects/NIST_TRIO/temp/VCF_Compare/${sample}/vcfStats.txt)"
```

Comment:
waarom loop je niet over deze map -->  /groups/umcg-gsad/tmp03/projects/NIST_TRIO/temp/VCF_Compare/${sample}/

Reply:
omdat ${sample} een variabele is die binnen de for loop aangemaakt wordt